ceph.spec.in

```diff
@@ -484,12 +484,16 @@ Requires:       python%{_python_buildid}-cherrypy
 Requires:       python%{_python_buildid}-jwt
 Requires:       python%{_python_buildid}-routes
 Requires:       python%{_python_buildid}-werkzeug
+%if 0%{?weak_deps}
+Recommends:     python%{_python_buildid}-saml
+%endif
 %endif
 %if 0%{?suse_version}
 Requires:       python%{_python_buildid}-CherryPy
 Requires:       python%{_python_buildid}-PyJWT
 Requires:       python%{_python_buildid}-Routes
 Requires:       python%{_python_buildid}-Werkzeug
+Recommends:     python%{_python_buildid}-python3-saml
```

Comment:
Please drop this line entirely - nautilus runs on SLE-15-SP1 which does not have this package.

Reply:
Done!